about/aml-kyc-requirements.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: AML/KYC Requirements
+excerpt: "What the Foundation is required to collect for compliance"
+---
+
+Last updated: December 2, 2020.
+
+
+_Compliance_:  **To execute a transfer of funds, the Zcash Foundation is legally required to obtain the following information from you: [Privacy guarantee]**
```

Comment:
Seems like there is a missing link to the Privacy guarantee here

Reply:
it's some legalese punctuation, it's not supposed to be a link